e2e/scenario_test.go

```diff
@@ -1071,6 +1071,35 @@ func runScenarioUbuntu2204GPU(t *testing.T, vmSize string) {
 	})
 }
 
+func runScenarioUbuntuGRID(t *testing.T, vmSize string) {
```

Comment:
should we keep 2204 in the ScenarioName?  what if we want to test with 2404 to ensure that it's working with the kernel 6.8 ?

Reply:
We should be checking both 22.04 and 24.04 in that case. How are we switching between the two OS versions?

Reply:
this line ? VHD:     config.VHDUbuntu2204Gen2Containerd,

Reply:
Thanks, added another test for 24.04

Reply:
Removed that test since there were other failures.